crates/nu-protocol/src/shell_error.rs

```diff
@@ -215,21 +215,11 @@ pub enum ShellError {
     /// Not all values can be coerced this way. Check the supported type(s) and try again.
     #[error("Can't convert to {0}.")]
     #[diagnostic(code(nu::shell::cant_convert), url(docsrs))]
```

Comment:
For folks reviewing: This is the most interesting part of this huge diff. Everything else is mechanical changes based on what's going on here.